### ISSUE TITLE
rpm-version string in copy-results gold file will eventually break unit tests

### ISSUE THREAD

**Issue by portante** [distributed-system-analysis/pbench#868]
Won't the `rpm-version` string in the `copy-results` [gold file](rpm-version string in copy-results gold file will eventually break unit tests) eventually break unit tests when the RPM version gets updated?

**Comment by ndokos** [comment]
No, it's just an artifact in the (supplied) metadata.log file and no modification is done to the `rpm-version` string: the only mods are adding a `user` and a `prefix` value to the run section and that's what the diff checks.

**Comment by ndokos** [comment]
After a discussion, we came up with the following plan: `pbench-{move,copy}-results` will check the `rpm-version` recorded in the metadata.log and if different from the current one, it will record the current one as well. That will provide a trail in  the metadata log for the scenario where somebody updates their pbench installation *after* producing some results but *before* moving them to the server.
